puppet/instrumental_tools/templates/instrumental.yml.erb

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We are introducing a fair number of new options in this release, but I don't think? we'll ever want to remove them. Like, I think the right philosophy to have here is that if a config file can specify it, it can also be specified via the command line.